### ISSUE TITLE
RGB light bulbs not changing to colour mode with Google Assistant or through the Google home app.

### ISSUE THREAD

**Issue by Phil-m-pinto** [leikoilja/ha-google-home#502]
## Version of the integration

If you are unsure about the version check the [github release here](https://github.com/leikoilja/ha-google-home/releases).

```Unable to change smart light bulbs to colour mode or change the colour of the bulb on the bulb is in white mode through Google Assistant or with the Google home app. However, I am able to change the colour through HA. Once the bulb is in colour mode Google then allows me to adjust its colour. But if I switch it back to white mode I can't switch it back to colour mode. This is an issue that has been reported in the tuya integration issues section. However, I am wondering if the problem is with the Google home integration  ( this is the link to the to your integration issue  https://github.com/home-assistant/core/issues/60476

Add version here.
```1.9.9

## Logs

<!-- To enable debug logs, put the below snippet in your configuration.yaml file.

logger:
  default: debug
  logs:
    custom_components.google_home: debug
-->

<!-- Paste logs below here-->

```text
Add your logs here.
```

**Comment by ArnyminerZ** [comment]
I don't see why this integration would have something to do with Tuya. This only controls Google Home devices, not any "domotics related devices". However, feel free to develop a bit more on the subject and we might be able to help.

**Comment by Phil-m-pinto** [comment]
> I don't see why this integration would have something to do with Tuya. This only controls Google Home devices, not any "domotics related devices". However, feel free to develop a bit more on the subject and we might be able to help.

Exactly. This is my thought. I don't think it is to do with the tuya integration. I think it maybe to do with the Google home integration.

**Comment by ArnyminerZ** [comment]
The thing is that this integration doesn't communicate with lights, even though Google Home does. This integration just interfaces with Google Home-inherited functionalities, not what GH is capable of. I mean, the integration just controls alarms, volume... Functions that "are in all the Google Homes", I don't know if I'm explaining myself.

This issue should be reported directly to the Tuya Integration [in HA core](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+tuya%22), or the integration in particular that controls your lights. Maybe Philips Hue, or whatever it is.

Hope this makes things clearer. If not, please ask again and I will try to help as much as possible.
